OrbitQt/orbitdataviewpanel.cpp

```diff
@@ -19,9 +19,12 @@ OrbitDataViewPanel::OrbitDataViewPanel(QWidget* parent)
 OrbitDataViewPanel::~OrbitDataViewPanel() { delete ui; }
 
 //-----------------------------------------------------------------------------
-void OrbitDataViewPanel::Initialize(DataViewType a_Type,
+void OrbitDataViewPanel::Initialize(DataView* data_view,
+                                    bool enable_extended_selection,
+                                    bool set_fixed_font,
                                     bool a_IsMainInstance) {
```

Comment:
Rename this parameter as the other parameters of this same method use another style?

Reply:
done.